### DISCUSSION TITLE
How to activate translations in production for React with i18next-http-backend ?

### DISCUSSION THREAD

**Discussion by soloman-rai** [i18next/i18next#1908] [Q&A]
Here's my current setup 

`
import i18n from "i18next";
import HttpApi from "i18next-http-backend";

import LanguageDetector from "i18next-browser-languagedetector";
import { initReactI18next } from "react-i18next";

i18n
  .use(initReactI18next)
  .use(LanguageDetector)
  .use(HttpApi)
  .init({
    fallbackLng: "en",

    debug: process.env.NODE_ENV === "development",

    ns: ["translations"],

    defaultNS: "translations",

    keySeparator: false, 

    interpolation: {
      escapeValue: false,

      formatSeparator: ",",
    },

    react: {
      wait: true,
    },
    backend: {
      loadPath: "/locales/{{lng}}/{{ns}}.json",
    },
  });

export default i18n;

`
It is working properly in development but not in production where I am serving the static files from build.  While inspecting the Network tab, I can see the message "You need to enable JavaScript to run this app."
Please provide some help, maybe specific pointers to make this work.

**Comment by jamuhl** [comment]
How should we help with only having a "You need to enable JavaScript to run this app." message and a few config settings...

First enabling javascript might help? Then when you get warnings for "not found" json files check the path to them is correct -> you can reach those files by using your browser
